RSVflights/src/screens/MyFlights.js

```diff
@@ -0,0 +1,74 @@
+import React, {useState, useEffect} from 'react';
+import {StyleSheet, SafeAreaView, StatusBar, Text, View, TouchableOpacity} from 'react-native';
+import Auth from '../components/Auth';
+import firebase from '../utils/firebase';
+import 'firebase/auth';
+import colors from '../utils/colors';
+
+
+export default function MyFlights(props) {
+  const {navigation} = props;
+  const [user, setUser] = useState(undefined);
+
+  useEffect(() => {
+    firebase.auth().onAuthStateChanged((response) => {
+      setUser(response);
+    });
+  }, []);
+
+  if (user === undefined) {
+    return null;
+  }
+  return (
+    <>
+      <StatusBar barStyle="light-content" />
+      <SafeAreaView style={styles.background}>
+        {user ? (
+          <View>
+            <Text style={styles.title}>Sing Up</Text> 
+            <Text>Logeado</Text>
+            <Text onPress={() => firebase.auth().signOut()}>Cerrar Sesión</Text>
```

Comment:
Add Sign up. Logeado, Cerrrar sessió to a Constant file

Reply:
Ok, thanks